c7n/resources/sqs.py

```diff
@@ -118,6 +119,28 @@ class SQSCrossAccount(CrossAccountAccessFilter):
     permissions = ('sqs:GetQueueAttributes',)
 
 
+@SQS.filter_registry.register('kms-alias')
+class KmsKeyAlias(ResourceKmsKeyAlias):
+    """
+    Filter a resource by its associated kms key aliasname 'AliasName'
+     :example:
+         .. code-block:: yaml
```

Comment:
getting a doc failure here because you need a new line before you start writing the example policy